oat-sa/tao-core
# Added check for mobile device

https://oat-sa.atlassian.net/browse/TAO-7845

This is a fix for the Calculator zoom behavior.

Steps to reproduce:
Invoke calculator > Zoom in > after typing in each number, control goes to the part where the result is displayed.

How to test on touch devices:

open https://act-test.taocloud.org
find delivery http://localhost/tao.rdf#i1503426381071581600217278 (just type this in the search box in the delivery) or anyone else with Calculator.
open delivery via lti (http://ltiapps.net/test/tc.php)
Invoke calculator > Zoom in > after typing in each number, control goes to the part where the result is displayed.
cause of the problem:
When you click on the numbers, the autofocus on the textarea is triggered and because of this, the screen takes the user to this textarea.

solution to the problem:
autofocus is disabled for mobile devices because it is not needed there.
